compass/landice/tests/antarctic/mesh.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+import netCDF4
+import xarray
+from matplotlib import pyplot as plt
+
+from mpas_tools.mesh.creation import build_planar_mesh
+from mpas_tools.mesh.conversion import convert, cull
+from mpas_tools.planar_hex import make_planar_hex_mesh
+from mpas_tools.io import write_netcdf
+from mpas_tools.logging import check_call
+
+from compass.step import Step
+from compass.model import make_graph_file
+from compass.landice.mesh import gridded_flood_fill, \
+                                 set_rectangular_geom_points_and_edges, \
+                                 set_cell_width, get_dist_to_edge_and_GL
+
+
+class Mesh(Step):
+    """
+    A step for creating a mesh and initial condition for antarctica test cases
+
+    Attributes
+    ----------
+    mesh_type : str
+        The resolution or mesh type of the test case
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        mesh_type : str
+            The resolution or mesh type of the test case
+        """
+        super().__init__(test_case=test_case, name='mesh')
+
+        self.add_output_file(filename='graph.info')
+        self.add_output_file(filename='Antarctic_1to10km.nc')
+        self.add_input_file(
+                filename='antarctica_8km_2020_10_20.nc',
```

Comment:
Just a question for everyone, is the 8 km data set good enough for this purpose or would you want to use something down to 1 km since the target mesh is higher res?  I assume this is BedMachine topography, which I seem to recall is available at 500 m resolution.

Reply:
We will very likely end up using a 1km dataset eventually, but this makes testing much faster.